### PR TITLE
merge list_objects_v2() into list_objects() method

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

**Comment by harshavardhana** [comment]
> LGTM, although this looks like a breaking change, maybe we need to increase major version number now

Yes, unfortunately, we will have to do that @vadmeste

File: tests/unit/list_objects_test.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # MinIO Python Library for Amazon S3 Compatible Cloud Storage,
-# (C) 2015, 2016 MinIO, Inc.
+# (C) 2015 MinIO, Inc.
```
Comment (kannappanr) [inline tests/unit/list_objects_test.py:3]:
```suggestion
# (C) 2015-2020 MinIO, Inc.
```

**Comment by balamurugana** [comment]
Depends on #966

File: examples/list_objects.py [hunk 2]
```diff
@@ -32,8 +32,8 @@
 
 # List all object paths in bucket that begin with my-prefixname using
 # V2 listing API.
-objects = client.list_objects_v2('my-bucketname', prefix='my-prefixname',
-                                 recursive=True)
+objects = client.list_objects('my-bucketname', prefix='my-prefixname',
```
Comment (vadmeste) [inline examples/list_objects.py:35]: This example has duplicated code after your change, I guess you need to use api v1 in the first line and update some wording

**Review by vadmeste — COMMENTED** [review]
One comment, LGTM otherwise

File: examples/list_objects.py (hunk 2 above)
Comment (balamurugana) [reply examples/list_objects.py:35]: done

**Review by vadmeste — APPROVED** [review]
LGTM & tested

**Pull request merged**